tests/Component/RequestBuilder/EDisMaxTest.php

```diff
@@ -31,6 +31,7 @@ public function testBuildComponent()
         $component->setBoostFunctions('functionX(price)');
         $component->setBoostFunctionsMult('functionX(date)');
         $component->setUserFields('title *_s');
+        $component->setSplitOnWhitespace(false);
```

Comment:
Same as above. The test should be moved to Solarium\Tests\QueryType\Select\RequestBuilderTest